src/components/upload-started/upload-started-slide.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import ProgressBar from "../shared/progress-bar";
+import Slide from "../shared/slide";
+import Spinner from "../shared/spinner";
+
+const ICON_UPLOAD = require("../../assets/images/icon_upload.png");
+
+const UploadStartedSlide = ({ chunksProgress }) => {
+  const description =
+    chunksProgress >= 99.999 // epsilon b/c float comparison.
+      ? "Confirming upload..."
+      : "File is being broken into chunks and each chunk encrypted...";
+
+  return (
+    <Slide title="Upload Started" image={ICON_UPLOAD}>
+      <p className="transaction-confirmed-instructions">
```

Comment:
styled components

Reply:
I'm just reusing this class. Can we do the conversion in a separate PR, this one is already pretty big